app/assets/javascripts/training/components/training_slide_handler.jsx

```diff
@@ -195,6 +196,9 @@ const TrainingSlideHandler = createReactClass({
       }
     }
 
+    // setting the page title based on the slide
+    document.title = `${slideTitle} - ${this.baseTitle}`;
```

Comment:
Since TrainingSlideHandler needs to update the title whenever the slide changes, rather than just when the component first mounts, this component might be better handled with `useEffect`.

Reply:
So as we discussed, I'll convert this to a FC and then implement the dynamic titles